tests/zfs-tests/include/libtest.shlib

```diff
@@ -2843,3 +2843,30 @@ function block_device_wait
 		$UDEVADM settle
 	fi
 }
+
+function kill_process
+{
+	typeset -l i killpid=$1
+
+	#attempt to kill
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_must $KILL $killpid
+	fi
+	#verify killed worked
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_mustnot $KILL $killpid
+	fi
+}
+
+function start_io
+{
+	typeset -l i sizetouse=$1
+	typeset -l i blocksize=$2
+	typeset -l i filename=$3
+
+	#start some IO
+	#This used to be: $FILE_TRUNC -f $((64 * 1024 * 1024)) -b 8192 -c 0 -r $TESTDIR/$TESTFILE1 &
+	dd_string="$DD if=/dev/urandom iflag=fullblock bs=$blocksize count=$sizetouse of=$filename & "
+	log_note "$dd_string"
+	dd_string
```

Comment:
Yes, we should use better function names.  Yes, log_must makes since for starting a command. 

Do you think FILE_TRUNC is the best tool for an IO load?  We can make a dd command line that works on Linux, Darwin, and freebsd.  But we can't for FILE_TRUNC.  On Linux it returns too quickly we want an IO load, not something that returns quickly.   

We looked at other options like shred but they did not prove to be portable to other platforms.  My suggestion was to write some pattern IO and we could use dd or not.